packages/tasks/src/tasks/text-to-image/about.md

```diff
@@ -16,9 +16,23 @@ Different patterns can be generated to obtain unique pieces of fashion. Text-to-
 
 Architects can utilise the models to construct an environment based out on the requirements of the floor plan. This can also include the furniture that has to be placed in that environment.
 
-## Task Variants
+##Task Variants
 
-You can contribute variants of this task [here](https://github.com/huggingface/hub-docs/blob/main/tasks/src/text-to-image/about.md).
+### Image Editing
+
+Image editing with text-to-image models involves using text prompts to describe the wanted changes in an image and then follow them.
+
+- Synthetic image editing: using text-to-image models to make adjustments to images that were initially created using an input prompt, while preserving the overall meaning or context of the original image.
+
+  ![Examples](https://datasets-server.huggingface.co/assets/diffusers/diffusers-images-docs/--/b20ecaa3f61372174c854e09fc856fdcce6f8494/--/default/train/0/image/image.png?Expires=1725455983&Signature=ykj3EnAENI6goXc7qI2Toq~P8P5IdS1DqNbSfH8vhgrdwaJoGH2cUbXWRgVAndhrHvRjrTTcU3YOyoExnot7zEhauyUEcqr-evRHDmGgfar52uEmfLbLCtNAcRK9Q85QOifupIH-X9x3rBUM03B0RIkHuto6wwRBAHireqr7QcD8hYRaNzACXrTbt-U7wHosZS8R1pdc3FDt7fDc3Qwh8XL0YoJqAoK8X8JnZEXIWTfGnCpygPBDbseDlYEzegGKzClAUgigQbomUk733VNtB3ol396uYkHCcjqjtgdhtEfAWQz-xM4eAhHpI~YEn7RQqRjB0RD0bPd1nHRU0wGUqA__&Key-Pair-Id=K3EI6M078Z3AC3)
+
+- Real image editing: similar to synthetic image editing, except we're using real photos/images. This task is usually more complex, as it involves first obtaining a latent representation of the image, in the latent domain of the model that it can then manipulate.
+
+  ![Examples](https://datasets-server.huggingface.co/assets/diffusers/diffusers-images-docs/--/default/train/1/image/image.jpg?Expires=1725453082&Signature=MOCeELTChydgLRZT9ws8owCraSVrdcm6c7Vlnsi23rJ1Ocigl6gjRtXwmjVDCKuG2fB6Hw0Tmn8ZR0M7FPiA2fXpSuPEW4iJMoeQNiNCtkSSjjDisDXbBSRXW1TXJ-Z2c~VoJ4lmmeUdFpyFZ9W~BlI6r2xQLltfU400XKPe~UgE-vJ~xr9ni8zZmyYt1kVtV9Et~EBzWCQkKc2DO9gI9HnEg9z2hxDHp8Bak0HBRARM4ObhRYxieWqO4hOg1HVk4LSt2E8emIuDmhPUU4v8L097yFcI4D6JeoyNNn0q6nKQZqAZIzwP8iiLqqhSv~mJsO7YGnQck1-bzA~gAiVMpg__&Key-Pair-Id=K3EI6M078Z3AC3)
+
+### Personalization
+
+Personalization refers to techniques used to customize text-to-image models, where we introduce new subjects/concepts to the model so that we can then use the model to generate new images of those subjects with a text prompt. For example, one can use these techniques to generate images of themselves, using as little as one reference image. These include teaching the model a new concept both in training free manner or through fine-tuning.
```

Comment:
```suggestion
Personalization refers to techniques used to customize text-to-image models. In this technique, we introduce new subjects/concepts to the model and then use the model to generate new images of those subjects with a text prompt. 

For example, one can use these techniques to generate images of themselves, using as little as one reference image. These include teaching the model a new concept both in a training-free manner or through fine-tuning.
```

I am actually irritated by personal DreamBooths, to be honest, especially with the recent Korean cases where it's weaponized against women. Can you change the example?

Reply:
That's a good point. But I also feel like it's such a major use case, it's a bit odd not to mention it. Cause mentioning style Loras can also potentially be problematic on the artists angle

Reply:
perhaps mentioning both?

Reply:
These are good points. A couple of alternatives:
- Use "your dog" as an example (I did this in my suggestion). I agree with Linoy that this somewhat sweeps a major use case under the rug.
- Be open about it and include a comment about using these features responsibly, respectfully, and ethically.